public/locales/en/one-time-donation.json

```diff
@@ -52,7 +52,7 @@
     "card": "Card",
     "card-include-fees": "I want to cover transaction fees for card issued in:",
     "card-fees": "For donations by card we use the services of Stripe and for every transfer they charge a fee depending on region of your card issuer. To orient you about the net donation, after you choose the desired amount, we will show you the calculated fee below. More details for Stripe fees can be found here: ",
-    "card-calculated-fees": "For your donation of <strong>{{amount}}</strong>, the fee from Stripe will be <strong>{{fees}}</strong>, and the total charged amount will be <strong>{{totalAmount}}</strong>",
+    "card-calculated-fees": "For your donation of <strong>{{totalAmount}}</strong>, the fee from Stripe will be <strong>{{fees}}</strong>, and the total charged amount will be <strong>{{amount}}</strong>",
```

Comment:
All good with the swap of fields, however we also need to make the text to be literally the same like the BG translation. So it should be like this - see the words in itallic:
"For your _transfer_ of {{totalAmount}} the fee from Stripe will be {{fees}} and _the campaign will receive_ {{amount}}".

Reply:
Done.